### PR TITLE
Update starters.yml with new entry

### PR DESCRIPTION
**Pull request by damassi**
Adds a link to a new starter, https://github.com/damassi/gatsby-starter-typescript-rebass-netlifycms.

### CODE REVIEW

**Comment by damassi** [comment]
If anyone has any advice on how to get this pass the linting stage :) It's been run through prettier, so I'm not sure what the issue is.

File: docs/starters.yml [hunk 1]
```diff
@@ -1710,3 +1710,25 @@
...
+- url: https://frosty-ride-4ff3b9.netlify.com/
+  repo: https://github.com/damassi/gatsby-starter-typescript-rebass-netlifycms
+  description:
+    A Gatsby starter built on top of MDX (React + Markdown), NetlifyCMS (with
+    MDX and netlify-cms-backend-fs support -- no need to deploy), TypeScript,
+    Rebass for UI, Styled Components, and Jest for testing. Very little visual
+    styling has been applied so that you can bring your own :)
+  tags:
+    - MDX
+    - NetlifyCMS
```
Comment (LekoArts) [inline docs/starters.yml]:
```suggestion
    - Netlify CMS
```

File: docs/starters.yml [hunk 2]
```diff
@@ -1710,3 +1710,25 @@
...
+    - netlify-cms-backend-fs
```
Comment (LekoArts) [inline docs/starters.yml]: Please remove that

```suggestion
```

File: docs/starters.yml [hunk 3]
```diff
@@ -1710,3 +1710,25 @@
...
+    - TypeScript
+    - Rebass
```
Comment (LekoArts) [inline docs/starters.yml]:
```suggestion
    - Styling:Rebass
```

File: docs/starters.yml [hunk 4]
```diff
@@ -1710,3 +1710,25 @@
...
+    - styled-components
```
Comment (LekoArts) [inline docs/starters.yml]:
```suggestion
    - Styling:CSS-in-JS
```

File: docs/starters.yml [hunk 5]
```diff
@@ -1710,3 +1710,25 @@
...
+  features:
+    - MDX - Markdown + React
+    - NetlifyCMS (with MDX support)
```
Comment (LekoArts) [inline docs/starters.yml]:
```suggestion
    - Netlify CMS (with MDX support)
```

**Comment by damassi** [comment]
Updated 👍

**Pull request merged**